### PR TITLE
Bump Hecke to 0.13

### PR DESCRIPTION
**Pull request by thofma**
Includes #1127

### CODE REVIEW

**Comment by thofma** [comment]
@StevellM: This (https://github.com/oscar-system/Oscar.jl/pull/1131/files#diff-f7cb2ad48cb87520d5b6d47e4ddf73200a59e58754a910a200206f8790c777d3) is correct, right?

**Comment by StevellM** [comment]
Good, thank you for this: yes it seems correct!

File: Project.toml [hunk 1]
```diff
@@ -23,7 +23,7 @@ msolve_jll = "6d01cc9a-e8f6-580e-8c54-544227e08205"
 AbstractAlgebra = "0.24.1"
 DocStringExtensions = "0.8"
 GAP = "0.7.7"
-Hecke = "0.11.1"
+Hecke = "0.12.1"
```
Comment (fingolfin) [inline Project.toml]: There is no 0.12.1 yet, though? Perhaps you meant this?
```suggestion
Hecke = "0.12.0"
```

**Comment by thofma** [comment]
Help @joschmitt someone is using `echelon!`

**Comment by joschmitt** [comment]
Oh, sorry. Should I just push the fix to the `push_row` branch?

**Comment by thofma** [comment]
Yes, please (just add a commit, not a force push)

**Comment by joschmitt** [comment]
Done.

**Pull request merged**